CHANGES.rst

```diff
@@ -1,7 +1,47 @@
 Changes
 *******
 
+0.6.0 (2020-06-25)
+==================
+
+* Add split_merged_climos process
+* Update demo notebooks
+* Add decompose_flow_vectors process
+
+0.5.1 (2020-06-23)
+==================
+
+* Standardized code format with black
+* Refactored processes to use utils
+* Update notebook demos
+* Fix instability with online tests
+* Update ce-dataprep to 0.8.2
```

Comment:
This should be 0.8.3.

Reply:
Good catch. At the time of that release `0.8.2` was the most up-to-date version.  Since then though we have updated it so I have added it to the `0.6.0` release notes.